cfme/common/provider_views.py

```diff
@@ -580,10 +580,8 @@ def before_fill(self, values):
                 getattr(self, widget).fill(values.get(widget))
 
 
-class TemplatesCompareView(InfraProvidersView):
+class TemplatesCompareView(CompareView):
```

Comment:
The `verify_checked_items_compared`  method that was written for this class can be moved to the common view class. Can be in a separate PR.

Reply:
That method needs some fixing anyway, it shouldn't be calling pytest.fail directly, and should be referencing `self` instead of taking a view instance as an arg.

Reply:
I made a note and will be fixing in forthcoming PR.